WRITING_DOCS.md

```diff
@@ -9,8 +9,6 @@
   - [Divider](#divider)
   - [Grid](#grid)
   - [Mini-Card](#mini-card)
-  - [Dark-Mode](#dark-mode)
-  - [Light-Mode](#light-mode)
```

Comment:
The hover over for the Slice1 & Slice2 selectors seems broken.

Previously it looked like:
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/037a9df5-db32-403e-84ea-81dc9255519f)
And when your cursor moves into the bubble, the bubble stayed (good).

Now it looks like this with some weird line:
![image](https://github.com/icerpc/icerpc-docs/assets/25963603/c239a0a6-f272-4e0f-bd8d-6795c2a21fa2)
And once your cursor leaves the Slice1 button, the popup disappears. This makes it impossible to click "Learn More".

Reply:
Also the new bubble is larger, you can see in one picture it completely covers "Overview", but in the first picture it only partially covered overview.

I think the original (smaller) bubble is better, since it doesn't hide as much of the side-nav.

Reply:
Fixed the padding but I have no idea what is going on with that line. I didn't really touch that code. May have been an issue from updating the dependencies.